src/containers/dashboard/Dashboard.js

```diff
@@ -94,58 +128,81 @@ const Dashboard = ({ match, organizationId, root } :Props) => {
 
   return (
     <>
-      <StatsGridRow>
-        <Card>
-          <CardSegment>
-            <Typography variant="h3">{TOTAL_CASES}</Typography>
-            <Typography variant="h1">{casesStats.get(TOTAL_CASES, 0)}</Typography>
-          </CardSegment>
-        </Card>
-        <Card>
-          <CardSegment>
-            <Typography variant="h3">{OPEN_CASES}</Typography>
-            <Typography variant="h1">{casesStats.get(OPEN_CASES, 0)}</Typography>
-          </CardSegment>
-        </Card>
-        <Card>
-          <CardSegment>
-            <Typography variant="h3">{SUCCESSFUL}</Typography>
-            <Typography variant="h1">{casesStats.get(SUCCESSFUL, 0)}</Typography>
-          </CardSegment>
-        </Card>
-      </StatsGridRow>
-      <StatsGridRow>
-        <Card>
-          <CardSegment>
-            <Typography variant="h3">{TERMINATED}</Typography>
-            <Typography variant="h1">{casesStats.get(TERMINATED, 0)}</Typography>
-          </CardSegment>
-        </Card>
-        <Card>
-          <CardSegment>
-            <Typography variant="h3">{RESPONDENT_DECLINED}</Typography>
-            <Typography variant="h1">{casesStats.get(RESPONDENT_DECLINED, 0)}</Typography>
-          </CardSegment>
-        </Card>
-        <Card>
-          <CardSegment>
-            <Typography variant="h3">{NO_CONTACT}</Typography>
-            <Typography variant="h1">{casesStats.get(NO_CONTACT, 0)}</Typography>
-          </CardSegment>
-        </Card>
-      </StatsGridRow>
-      <Card>
-        <TableHeader>Staff Cases</TableHeader>
-        <CardSegment padding="0">
-          <Table
-              data={staffCasesData.toJS()}
-              headers={staffCasesTableHeaders}
-              isLoading={isPending(fetchRequestState)} />
-        </CardSegment>
-      </Card>
-      <ButtonWrapper>
-        <Button onClick={() => openStaffModal(true)} variant="text" mode="primary">+Add Staff Member</Button>
-      </ButtonWrapper>
+      <NavButtonGrid>
+        <NavButton
+            onClick={() => setVisibleScreen(STATS)}
+            selected={visibleScreen === STATS}>
+          {STATS}
+        </NavButton>
+        <NavButton
+            onClick={() => setVisibleScreen(SEARCH_CASES)}
+            selected={visibleScreen === SEARCH_CASES}>
+          {SEARCH_CASES}
+        </NavButton>
```

Comment:
Since you're not navigating, these are not actually nav buttons, which is even more of a reason to use MUI button.